zebra-consensus/src/checkpoint/tests.rs

```diff
@@ -313,22 +306,28 @@ async fn continuous_blockchain(restart_height: Option<block::Height>) -> Result<
 
         // Now verify each block
         for (block, height, _hash) in blockchain {
+            // Commit directly to the state until after the (fake) restart height
             if let Some(restart_height) = restart_height {
                 if height <= restart_height {
                     let mut state_service = state_service.clone();
                     /// SPANDOC: Make sure the state service is ready for block {?height}
                     let ready_state_service =
                         state_service.ready_and().map_err(|e| eyre!(e)).await?;
 
-                    /// SPANDOC: Add block to the state {?height}
+                    /// SPANDOC: Add block directly to the state {?height}
                     ready_state_service
                         .call(zebra_state::Request::CommitFinalizedBlock(
                             block.clone().into(),
                         ))
                         .await
                         .map_err(|e| eyre!(e))?;
+
+                    // Skip verification for (fake) previous blocks
+                    continue;
```

Comment:
This was the bug stopping us from re-enabling these tests @dconnolly 😮

Reply:
😮